src/Sentry/Protocol/ITransaction.cs

```diff
@@ -18,6 +18,12 @@ public interface ITransaction : ISpan, ITransactionContext, IEventLike
         // 'new' because it adds a setter
         new string Name { get; set; }
 
+        // Should this be on IEventLike?
```

Comment:
Yeah we should bump this so it's available on any Event (transaction or errors)

Reply:
The problem is currently `IEventLike` also encompasses `Scope`, which does not have `Release` :/

Reply:
We can add it, no problem